src/state/features/userSlice.ts

```diff
@@ -0,0 +1,34 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
+
+type userInfoType = {
+  name: string
+  email: string
+  img?: string
+  isLoggedIn: boolean
+}
+
+const initialState: userInfoType = {
+  name: '',
+  email: '',
```

Comment:
```suggestion
  email: '',
  img: '',
```
make it consistent with logout